app/src/main/java/de/danoeh/antennapod/fragment/ExternalPlayerFragment.java

```diff
@@ -206,9 +205,9 @@ private void updateUi(Playable media) {
                 .dontAnimate();
 
         Glide.with(getActivity())
-                .load(ImageResourceUtils.getEpisodeListImageLocation(media))
+                .load(media.getImageLocation())
                 .error(Glide.with(getActivity())
-                        .load(ImageResourceUtils.getFallbackImageLocation(media))
+                        .load(media.getImageLocation())
```

Comment:
Something is wrong here. Image and its fallback are the same.

Reply:
Yes, see comment above.